go/vt/worker/legacy_split_clone.go

```diff
@@ -531,6 +546,14 @@ func (scw *LegacySplitCloneWorker) copy(ctx context.Context) error {
 		if err != nil {
 			return vterrors.Wrapf(err, "cannot build vschema for keyspace %v", scw.keyspace)
 		}
+
+		_vc, cleanupVCursor, err := vcursor.NewVCursor(ctx, scw.vcursorArgs)
+		if err != nil {
+			return fmt.Errorf("could not create vcursor: %v", err)
+		}
+
+		vc = _vc
+		defer cleanupVCursor()
```

Comment:
depending on how it performs in error cases you might want this before the err check / to always run

Alternatively (likely) on error we fail so it doesn't matter

Reply:
that's a good callout! updated

Reply:
actually, have to roll this back since a non-nil error means that this function will be nil and cause null pointer references when running tests :-D